src/components/molecules/UserProfilePicture/UserReactions.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useMemo } from "react";
+import classNames from "classnames";
+
+import {
+  isTextReaction,
+  ReactionData,
+  EmojiReactionType,
+} from "types/reactions";
+import { User } from "types/User";
+
+import { WithId } from "utils/id";
+import { uniqueEmojiReactionsDataMapReducer } from "utils/reactions";
+
+import { useReactions } from "hooks/useReactions";
+import { useSelector } from "hooks/useSelector";
+
+import "./UserReactions.scss";
+
+export interface UserReactionsProps {
+  venueId: string;
+  user: WithId<User>;
+  isMuted?: boolean;
+  reactionPosition?: "left" | "right";
+}
+
+export const UserReactions: React.FC<UserReactionsProps> = ({
+  venueId,
+  user,
+  isMuted: isMutedLocally = false,
+  reactionPosition,
+  children,
+}) => {
+  // @debt some of the redux patterns exist for this, but I don't believe anything actually uses them/calls this at the moment. Used in MapPartygoersOverlay
+  const isMutedGlobally = useSelector((state) => state.room.mute);
+  const isMuted = isMutedLocally || isMutedGlobally;
+
+  const userReactions = useReactions({ venueId, user });
+
+  const { renderedEmojiReactions, userShoutout } = useMemo(() => {
+    const userUniqueEmojiReactions = userReactions.reduce(
+      uniqueEmojiReactionsDataMapReducer,
+      new Map()
+    );
+
+    const renderedEmojiReactions = Array.from(
+      userUniqueEmojiReactions.values()
+    ).map((emojiReaction) => (
+      <DisplayEmojiReaction
+        key={emojiReaction.type}
+        emojiReaction={emojiReaction}
+        isMuted={isMuted}
+      />
+    ));
```

Comment:
Why do we not create an array in the reduce?

Reply:
I believe I'm using the `Map` as the way of making it unique values. If they were primitive values then I could have used `Set`, but I don't believe that will work as well given they're complex objects.

Given this is the only place that uses this/needs it as an array, it didn't seem like something that was worth further abstracting into another util function/etc.

Reply:
Marking this resolved based on the above.